doc/python/axes.md

```diff
@@ -492,15 +490,14 @@ fig.show()
 
 _new in 5.8_
 
-By default grid lines are `solid`. Set the `griddash` property to change this style. In this example we display the x-axis grid lines as `dot`. It can also be set to `dash`, `longdash`, `dashdot`, or `longdashdot`.
+By default grid lines are `solid`. Set the `griddash` property to change this style. In this example we display the x-axis grid lines as `dash` and the minor grid lines as `dot`. Other allowable values are `longdash`, `dashdot`, or `longdashdot`.
 
 ```python
 import plotly.express as px
 df = px.data.iris()
 
 fig = px.scatter(df, x="sepal_width", y="sepal_length", facet_col="species")
-fig.update_xaxes(showgrid=True, gridwidth=1, gridcolor='LightPink', griddash='dot')
-fig.update_yaxes(showgrid=True, gridwidth=1, gridcolor='LightPink')
+fig.update_xaxes(gridcolor='black', griddash='dash', minor_griddash="dot")
```

Comment:
the pink was not very attractive, and was hard to see IMO. also, `showgrid` is on by default in xaxes and `gridwidth=1` by default. I try not to explicitly set the defaults in the docs, so folks don't think it's necessary.